pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -20,19 +20,29 @@ import (
 
 func (r *ReconcilePerconaServerMongoDB) reconsileSSL(cr *api.PerconaServerMongoDB) error {
 	secretObj := corev1.Secret{}
-	err := r.client.Get(context.TODO(),
+	secretInternalObj := corev1.Secret{}
```

Comment:
@defbin please add good description to your PR as it was decided

Reply:
Sure, I have updated it in the description.

Reply:
Sure, I have updated it in the description.

Reply:
Yes I just added a description